PyInstaller/utils/hooks/__init__.py

```diff
@@ -488,8 +489,11 @@ class method (e.g., `idontevenknow<1.6,>1.9,!=1.9.6,<2.0a0,==2.4c1`) is
     # If no version was explicitly passed, query this module for it.
     if version is None:
         module_name = requirements_parsed.project_name
-        version = get_module_attribute(module_name, version_attr)
-
+        try:
+            version = get_module_attribute(module_name, version_attr)
+            print(version)
+        except ImportError:
+            raise
```

Comment:
No need for try/except here since the only thing you do in the except-clause is to raise the same exception again. Or did I miss something?

Also this changes the behavior of `is_module_satisfies` in an irritating way: If the module is missing and a `version` is *not* given, this will raise. Whereas if `version` is given, it will not raise.

Thus this part should read:
```
try:
    version = get_module_attribute(module_name, version_attr)
except ImportError:
    pass  ## or return False
```